.github/workflows/ci-ember-flight-icons.yml

```diff
@@ -32,7 +32,7 @@ name: CI - Ember Flight Icons
 
 defaults:
   run:
-    working-directory: ember-flight-icons
+    working-directory: @hashicorp/ember-flight-icons
```

Comment:
@Dhaulagiri At a stopping point for the day. This change kills CI, something to debug tomorrow. Even in some other testing that I scrapped, not sure how to handle the package name including a `/`.

Am almost wondering if a better next step would be to bring 2 packages into a new monorepo, e.g. design-system-components and design-system-tokens. 

If you have any thoughts just lmk

Reply:
TIL, thanks!

I'll find some examples of monorepos, and try to see if can debug the "/" in the package name

Reply:
can you say more about the `/` being a problem. is it a problem elsewhere besides the test here?

Reply:
![image](https://user-images.githubusercontent.com/1372946/149406098-0edf7b0b-8a4b-46a3-bc87-53ab07546618.png)

It seems solved here https://github.com/hashicorp/react-components#adding-package-dependencies but they're using npm workspaces and changesets. Still need to take a closer look into that example